### PR TITLE
Allow host base URI as an argument to ctor

### PR DESCRIPTION
**Pull request by cthrash**

### CODE REVIEW

File: ClientLibrary/FaceClientSDK.nuspec [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <description>Microsoft Project Oxford Face API Client Library</description>
+    <description>Microsoft Cognitive Services Face API Client Library</description>
     <summary>Use Face API Client Library to enrich your apps with Microsoft's state-of-the-art cloud-based face algorithms.</summary>
-    <releaseNotes>Project Oxford Face API V1.0 Client SDK</releaseNotes>
-    <copyright>Copyright 2015</copyright>
-    <tags>Face, Oxford, ProjectOxford</tags>
+    <releaseNotes>User-configurable hostname</releaseNotes>
+    <copyright>Copyright (c) 2016</copyright>
+    <tags>Face, Oxford, ProjectOxford Microsoft Cognitive</tags>
```
Comment (wangjun-microsoft) [inline ClientLibrary/FaceClientSDK.nuspec]: missing a "," after ProjectOxford ?

File: ClientLibrary/Microsoft.ProjectOxford.Face.csproj [hunk 2]
```diff
@@ -68,13 +68,13 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>packages\Newtonsoft.Json.6.0.8\lib\portable-net40+sl5+wp80+win8+wpa81\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>packages\Newtonsoft.Json.9.0.1\lib\portable-net45+wp80+win8+wpa81\Newtonsoft.Json.dll</HintPath>
```
Comment (wangjun-microsoft) [inline ClientLibrary/Microsoft.ProjectOxford.Face.csproj]: intentionally upgrade the dependency package?

File: ClientLibrary/FaceClientSDK.nuspec (hunk 1 above)
Comment (cthrash) [inline ClientLibrary/FaceClientSDK.nuspec]: The commas don't matter.  I've removed them

File: ClientLibrary/Microsoft.ProjectOxford.Face.csproj (hunk 2 above)
Comment (cthrash) [inline ClientLibrary/Microsoft.ProjectOxford.Face.csproj]: Yes.

**Pull request merged**